app/Service/StudentDetailsService.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use App\Exceptions\StudentNotFoundException;
+use App\Models\Resume;
+use App\Models\Student;
+use App\Models\Tag;
+
+class StudentDetailsService
+{
+    public function execute($student)
+    {
+        return $this->getStudentDetailsById($student);
+    }
+    
+
+    public function getStudentDetailsById($student){
+        
+        $studentDetails = Resume::where('student_id', $student)->first();
```

Comment:
In this case, I think it would be more appropriate to use the name ``$resume`` instead of ``$studentDetails``, since what the variable is really storing is the resume of a student.